torch/csrc/jit/python_ir.cpp

```diff
@@ -68,6 +68,35 @@ std::ostream& printPyObject(std::ostream& out, const THPObjectPtr& obj) {
   }
 }
 
+std::vector<Node*> findAllNodes(Block* block, Symbol kind) {
```

Comment:
I don't think these are really necessary to add if it's just for the test, this functionality can be done per-case via the Python bindings for `Graph` and `Node`

Reply:
What do you mean per-case via the bindings for Graph and Node? 

These will also be used in other tests, and have been a common pattern I've needed & couldn't easily express so I used expect files.

Reply:
If I don't add it as a method to Graph / Node then you can't chain method calls, so something like the following becomes difficult to write:
`
self.assertTrue(graph.findNode("prim::Loop").outputsSize() == 2)
`